CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## 6.2.2
+## 6.2.2(Upcoming Release)
```

Comment:
6.2.2 is already released, now it's 6.2.3

Reply:
I changed it